examples/iOS/SimpleImageFilter/SimpleImageFilter/Kirakira_Glamour.json

```diff
@@ -74,5 +74,13 @@
     "frameRate" : {
         "type" : "int",
         "value" : 60
-    }
+    },
+    "maxLength" : {
```

Comment:
Should we update this to `targetDimension`?

Reply:
I think we can just use the default value